index.md

```diff
@@ -106,26 +106,20 @@ Here is a partial list of its features:
 
 :::{toctree}
 :hidden:
-:caption: About PyMC
 
 about/ecosystem
 about/history
 about/testimonials
-:::
-
-:::{toctree}
-:hidden:
-:caption: Blog
 
-Recent posts <blog>
-Blog feed <https://www.pymc.io/blog/atom.xml>
+Blog <blog>
 :::
 
 :::{toctree}
 :hidden:
 :caption: External links
 
 PyMC documentation <https://docs.pymc.io>
+PyMC latest docs <https://docs.pymc.io/en/latest>
```

Comment:
This would confuse me as a user. Should we have it say `PyMC3` and `PyMC 4.0`?

Reply:
we can remove this. At most this is temporal while stable (aka the "pymc documentation" link) points to v3 so that it is possible to make the trip back and forth easily. cc @reshamas